### PR TITLE
Fix planet to use Supabase, remove feedsUrl/userFeedsUrl from front-end

### LINKED ISSUE #3463: Use Supabase in planet service vs. posts/feeds, remove feedsUrl from front-end
We need to switch the planet service to use Supabase, see https://github.com/Seneca-CDOT/telescope/blob/master/src/api/planet/src/planet.js#L6.  We can also remove the FEEDS_URL from the frontend, which isn't used.

### PR DESCRIPTION
**Pull request by humphd**
Fixes #3463.  This updates the `planet` service to use Supabase for the feeds info vs. Redis (cc @TueeNguyen, you can steal my code for getting author names for the parser, and @DukeManh is going to turn this into a stored procedure in the future).

I've also removed the unneeded `feedsUrl` and `userFeedsUrl` from the front-end.

@JerryHue After this I'm not sure if anything is using `posts/feeds` anymore.

### CODE REVIEW

**Comment by humphd** [comment]
If you want to test this:

```
cd src/api/planet
pnpm dev
```

Now visit http://localhost:9876/planet

**Review by TueeNguyen — APPROVED** [review]

File: src/api/planet/src/util.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    ).then(getPostDataGrouped);
+  } catch (err) {
+    return logger.error({ err }, 'Error getting posts');
+  }
+};
+
+module.exports.getFeeds = async () => {
+  const { data, error } = await supabase
+    .from('feeds')
+    .select('wiki_author_name, url, telescope_profiles (display_name)');
```
[the remark this answers is not in the document]
Comment (DukeManh) [reply src/api/planet/src/util.js:78]: Ah, I know why it's null, `telescope_profiles` is not public for read so we'll need to use SERVICE_ROLE_KEY here.

**Comment by humphd** [comment]
Fixed to use the `SERVICE_ROLE_KEY`

File: src/api/planet/src/util.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+  if (error) {
+    logger.warn({ error });
+    throw Error(error.message, "can't fetch feeds from supabase");
+  }
+
+  return sort(
+    data.map((feed) => ({
+      // Prefer the a user's display name if present, fallback to wiki name otherwise
+      author: feed.display_name || feed.wiki_author_name,
```
Comment (DukeManh) [inline src/api/planet/src/util.js:88]:
```suggestion
      author: feed.telescope_profiles?.display_name || feed.wiki_author_name,
```

**Review by DukeManh — CHANGES_REQUESTED** [review]

File: src/api/planet/src/util.js (hunk 2 above)
Comment (humphd) [reply src/api/planet/src/util.js:88]: So I need to include the table name when I do this join, for the returned object?

**Review by DukeManh — APPROVED** [review]

**Review by JerryHue — APPROVED** [review]

**Pull request merged**